azure-pipelines.yml

```diff
@@ -114,6 +115,25 @@ stages:
         b2 toolset=${TOOLSET} cxxstd=${CXXSTD} ${B2_ARGS}
         popd
       displayName: Test
+    - bash: |
+        set -e
+        SOURCE=$PWD
+        if [[ $CXXSTD == "2a" ]]; then
+          CXXSTD="20"
+        fi
+        mkdir -p ../run
+        mkdir -p ../build
+        pushd ../build
+          cmake -DCMAKE_INSTALL_PREFIX=../run $SOURCE -DCMAKE_CXX_STANDARD=${CXXSTD}
```

Comment:
@grafikrobot  In this line, I want only a single value of ${CXXSTD} since CMake does not support multiple configurations. But the current script provides "11,14,17,2a/20". I am not familiar enough with azure-pipelines to get an array and also, I'm not sure how would you like to proceed on this. 

One option is to define a new variable `CMAKE_CXXSTD` which does what we want. But I don't like the duplication. Another solution is to run `b2` also in a single configuration mode (at least with respect to `CXXSTD` as well.